.gitignore

```diff
@@ -56,3 +56,4 @@ package-lock.json
 ./ui/.env
 
 .pytest_cache/
+/api/gcp.json
```

Comment:
What is this file?

Reply:
This is the file we use in env which we are ignoring